lib/common_test/doc/src/ct_hooks_chapter.xml

```diff
@@ -511,6 +511,25 @@ results(State) ->
         use another level either change the <c>default</c> handler level before
         starting common_test, or use the <seemfa marker="kernel:logger#set_handler_config/3">
         <c>logger:set_handler_config/3</c></seemfa> API.</p>
+        <p>This hook supports the following options:</p>
+        <taglist>
```

Comment:
Please change order in the taglist, so default `add` is described as 1st and `replace` as second item.